earthpy/spatial.py

```diff
@@ -457,20 +478,23 @@ def plot_rgb(arr, rgb=(0, 1, 2),
     return fig, ax
 
 
-
 def hist(arr,
          title=None,
          colors=["purple"],
          figsize=(12, 12), cols=2,
          bins=20):
     """
-    Plot histogram each layer in a raster stack converted into a numpy array for quick visualization.
+    Plot histogram for each layer in a numpy array.
 
     Parameters
     ----------
     arr: a n dimension numpy array
-    title: a list of title values that should either equal the number of bands or be empty, default = none
-    colors: a list of color values that should either equal the number of bands or be a single color, (purple = default)
+    title: str_clip
```

Comment:
Is this supposed to be `str_clip`?

Reply:
goodness that must have been atom autocomplete. Str.  fixed.